gammapy/estimators/points/lightcurve.py

```diff
@@ -133,14 +135,15 @@ def run(self, datasets):
                 parallel_datasets.append(datasets_to_fit)
 
         if self.n_jobs > 1:
-            with Pool(processes=self.n_jobs) as pool:
-                rows = pool.starmap(
-                    self.estimate_time_bin_flux,
-                    zip(
-                        parallel_datasets,
-                        repeat(dataset_names),
-                    ),
-                )
+            rows = parallel.run_multiprocessing(
```

Comment:
Maybe the lightcurve multiprocessing could be tested as well?

Reply:
fixed, the n_jobs argument was not set correctly